README.md

```diff
@@ -70,6 +70,8 @@ $csrf = $csrfToken->getValue();
 
 ## CSRF Tokens
 
+Package is [configured](./config/web.php) by default with synchronizer token and masked decorator. You can change that easily, depending on your needs.
```

Comment:
```suggestion
In case Yii framework is used along with config plugin, the package is [configured](./config/web.php) automatically to use synchronizer token and masked decorator. You can change that depending on your needs.
```